### DISCUSSION TITLE
Data Labeling

### DISCUSSION THREAD

**Discussion by JMCarmona10** [reaktoro/reaktoro#335] [Q&A]
### Description

Greetings, Allan

I believe i have an understanding of reaktoro and the mathematical model employed for resolving chemical equilibrium, my question revolves on the presentation of output results. I'm posting a relatively straightforward Reaktoro program. My objective is to demonstrate how I can designate a particular specie in the final output as a variable, without resorting to the conventional "print(state)" method for visualizing the results. For example, the molality of H2O, how do i assign a variable and print its value without the usual way. 

`from reaktoro import *

db = PhreeqcDatabase("phreeqc.dat")

solution = AqueousPhase("H2O H+ OH-")

system = ChemicalSystem(db, solution)

state = ChemicalState(system)
state.set("H2O", 1.0, "kg")

equilibrate(state)

print(state)`

![reaktoro_1111](https://github.com/reaktoro/reaktoro/assets/116467897/24f118ee-f1a6-40e2-80b7-29996dbd18a3)

### What is your operating system?

None

**Comment by allanleal** [comment]
For aqueous related properties, you should use `AqueousProps`:

~~~py
aprops = AqueousProps(state)

print(aprops.speciesMolality("H+"))
~~~

Please check https://reaktoro.org/api for Reaktoro's API so that you understand all classes and methods available for you. There are some tutorials in the website covering this.

Hope this helps.
